.docs/user-guide/installation.md

```diff
@@ -40,6 +40,20 @@ specify `stable`, see the additional version names below that are also valid.
 curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable
 ```
 
+#### Install a specific version
+
```

Comment:
Hi @cantbewong,

Please be consistent with the rest of the documentation and omit a blank line between a heading and the paragraph that follows it.